packages/node/src/retryHandler.ts

```diff
@@ -103,10 +103,22 @@ export class RetryHandler implements RetryClass {
     return prunedEvents;
   }
 
-  private _getPayload(events: readonly Event[]): Payload {
+  private _getPayloadOptions(): Partial<Payload> {
+    if (typeof this._options.minIdLength === 'number') {
+      return {
+        options: {
+          min_id_length: this._options.minIdLength,
+        },
+      };
+    }
+    return {};
+  }
+
+  protected _getPayload(events: readonly Event[]): Payload {
```

Comment:
Testing was the only reason this method was changed from `private -> protected`.  `TestRetry` class, which extends `RetryHandler`,  can now call `_getPayload()` internally, which made testing possible.  Since no other classes extend TestRetry this change should not alter the private nature of `_getPayload`.

Reply:
oh! I forgot that distinction. It might make sense to make more things protected if they make sense to be reusable (e.g. payload options)